activerecord/test/cases/migration_test.rb

```diff
@@ -942,10 +942,9 @@ def migrate(x)
         }.new
 
         migrator = ActiveRecord::Migrator.new(:up, [migration], @schema_migration, 100)
+        query = "SELECT query FROM pg_stat_activity WHERE datname = '#{ActiveRecord::Base.connection_db_config.database}' AND state = 'idle'"
```

Comment:
@eileencodes I can't fix this yet because I don't know what query is leaking in this test causing the count to change. Instead, I changed the test to compare the idle queries in `pg_stat_activity` first so that when it fails again I have more information.